app/api/comment/create/func.js

```diff
@@ -0,0 +1,25 @@
+const errors = require('@/errors')
+const app = require('@/app')
+
+module.exports = async function(context){
+  const Comment = app.models.comments
+  const Enrollment = app.models.enrollments
+
+  if(!context.body.enrollment) throw new errors.BadRequest(`Missing enrollment`)
+
+  if(!context.body.comment) throw new errors.BadRequest(`Missing comment`)
+
+  let enrollment = await Enrollment.findById(String(context.body.enrollment))
```

Comment:
Não entendo o uso de String, você teve algum problema com isso?

Reply:
Sim, NoSQL injection. Coloco isso pra garantir que o vagabundo não vai me passar um { enrollment: { $ne: 'qualquer coisa' }}
Talvez por definir no model que o campo é do tipo ObjectId, imagino que o mongoose vai dar um throw se não passar um, mas, de qualquer forma eu coloco pra garantir pq se for qualquer outro caso vai funcionar